setup.py

```diff
@@ -11,13 +11,15 @@
     cmdclass=versioneer.get_cmdclass(),
     description="HMC samplers in Aesara",
     url="http://github.com/aesara-devs/aehmc",
-    maintainer="Brandon T. Willard",
+    maintainer="Aesara Developers",
     maintainer_email="aesara.devs@gmail.com",
     packages=["aehmc"],
     install_requires=[
         "numpy>=1.18.1",
         "scipy>=1.4.0",
-        "aesara",
+        "numba",
```

Comment:
This was removed because it's not being directly used by this library and it causes `pip` to overwrite the Conda environment's NumPy, which can cause the issue you're seeing in CI.